hepcrawl/spiders/pos_spider.py

```diff
@@ -32,24 +32,17 @@ class POSSpider(Spider):
     """POS/Sissa crawler.
 
     Extracts from metadata:
-        * title
-        * article-id
-        * conf-acronym
-        * authors
-        * affiliations
-        * publication-date
-        * publisher
-        * license
-        * language
-        * link
+        todo:: be added...
 
     Example:
         ::
 
-            $ scrapy crawl PoS -a source_file=file://`pwd`/tests/responses/pos/sample_pos_record.xml
+            $ scrapy crawl PoS -a source_file=file://`pwd`/tests/unit/responses/pos/
+            sample_pos_record.xml
     """
     name = 'PoS'
-    pos_base_url = "https://pos.sissa.it/contribution?id="
+    conference_paper_url = "https://pos.sissa.it/contribution?id="
```

Comment:
And all caps... though that is not used other other stuff here, so maybe leave the capitalization for a future refactor.